devise-two-factor.gemspec

```diff
@@ -28,7 +28,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency 'activesupport'
   s.add_runtime_dependency 'activemodel'
   s.add_runtime_dependency 'attr_encrypted', '~> 1.3.2'
-  s.add_runtime_dependency 'devise',         '>= 3.2.4', '< 3.5'
+  s.add_runtime_dependency 'devise',         '>= 3.2.4'
```

Comment:
I think we need the minimum devise version to be 3.5.0 with this change (Devise::Encryptor was introduced in https://github.com/plataformatec/devise/commit/a8769933d1842e8dff1a625d202f693403663e12)